python/sglang/srt/disaggregation/mooncake/conn.py

```diff
@@ -328,11 +348,44 @@ def __init__(
         self.bootstrap_room = bootstrap_room
         self.kv_mgr.set_status(bootstrap_room, KVPoll.WaitingForInput)
         self.aux_index = None
+        self.bootstrap_server_url = bootstrap_addr
+
+        self.session_id = self.kv_mgr.get_session_id()
+
+    def _register_to_bootstrap(self):
+        """Register KVSender to bootstrap server via HTTP POST."""
+        url = f"http://{self.bootstrap_server_url}/kv_route"
+        payload = {
+            "identity": self.session_id,
+            "role": "Prefill",
+            "serve_ip": self.kv_mgr.get_localhost(),
+            "serve_port": self.kv_mgr.rank_port,
+            "tp_rank": self.kv_mgr.kv_args.engine_rank,
+        }
+
+        logger.info(
+            f"Register prefill server port {self.kv_mgr.rank_port} for tp_rank {self.kv_mgr.kv_args.engine_rank}"
+        )
+
+        try:
+            response = requests.put(url, json=payload)
+            if response.status_code == 200:
+                logger.info(f"Prefill successfully registered to bootstrap server.")
+            else:
+                logger.info(
+                    f"Prefill Failed to register to bootstrap server: {response.status_code}, {response.text}"
+                )
+        except Exception as e:
+            logger.info(f"Prefill Failed to register to bootstrap server: {e}")
 
     def init(self, num_kv_indices: int, aux_index: Optional[int] = None):
         self.aux_index = aux_index
         self.num_kv_indices = num_kv_indices
 
+        # Register to bootstrap server
+        logger.info(f"Prefill bootstrap addr {self.bootstrap_server_url}.")
+        self._register_to_bootstrap()
```

Comment:
i think it's should be a class method , and need be cached  when registering the same prefiil node to bootstrap....